scripts/spell-check.sh

```diff
@@ -16,7 +16,7 @@
 #
 # By default, a hand-picked subset of Squid repository sources is fixed.
 #
-# See ${WHITE_LIST} below for the list of allowed misspellings.
+# See ${ALLOW_LIST} below for the list of allowed misspellings.
```

Comment:
This list should have been named IGNORE_LIST matching the codespell documentation. It is passed to the "--ignore-words" interface.